### PR TITLE
style: add size variations to badge

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added some size variation to badges

### CODE REVIEW

File: component/badge/badge.css [hunk 1]
```diff
@@ -39,6 +40,21 @@
...
+}
+
+.badge-round-lg {
+  height: 1.12rem;
+  width: 1.12rem;
+}
+
 .badge-sm {
   font-size: 12px;
   border-radius: 0.2rem;
```
Comment (parvez-rashid) [inline component/badge/badge.css]: Use px here since it is below 1rem.

File: component/badge/badge.css [hunk 2]
```diff
@@ -39,6 +40,21 @@
...
+  width: 1rem;
+}
+
+.badge-round-lg {
+  height: 1.12rem;
+  width: 1.12rem;
+}
+
 .badge-sm {
   font-size: 12px;
```
Comment (parvez-rashid) [inline component/badge/badge.css:59]: Use rem unless you have a purpose for px.

**Review by parvez-rashid — APPROVED** [review]
I have a few comments below, have a look. Other than, the code is neat and well-structured, can be merged.

File: component/input/input.css [hunk 3]
```diff
@@ -4,7 +4,7 @@
   padding: 8px 1rem;
   border-radius: 0.4rem;
   border: 3px solid rgba(114, 114, 114, 0.404);
```
Comment (yaswanthmyneni) [inline component/input/input.css:6]: changes: use css variables for color.

File: index.html [hunk 4]
```diff
@@ -358,20 +367,84 @@ <h1 class="component-title">Badge</h1>
...
+              style="
+                width: 680px;
+                height: 282px;
+                border: 0;
+                transform: scale(1);
```
Comment (yaswanthmyneni) [inline index.html]: changes: can make that inline style to a class and use it for iframes.

**Review by yaswanthmyneni — APPROVED** [review]
UI looks good, you can merge after changes.

File: component/badge/badge.css (hunk 1 above)
Comment (Sahil25061999) [reply component/badge/badge.css:60]: i am using rem if the value is greater or equal to 16px

File: component/badge/badge.css (hunk 2 above)
Comment (Sahil25061999) [reply component/badge/badge.css:59]: i am using rem if the value is greater or equal to 16px

**Pull request merged**